frontend/src/containers/User/AccountSetting/AccountSetting.js

```diff
@@ -145,7 +145,7 @@ class AccountSetting extends Component {
                     <input
                       id="editUserName"
                       type="text"
-                      value={this.state.username}
+                      value={this.state.username || ""}
```

Comment:
What is meaning of this part?

Reply:
When I didn't add this part, test warning occurred. (A component is changing a controlled input of type text to be uncontrolled.)

Reply:
So I follow the solution from here: https://stackoverflow.com/questions/47012169/a-component-is-changing-an-uncontrolled-input-of-type-text-to-be-controlled-erro

Reply:
Ok. When I saw those errors, mostly it is due to ‘undefined’ values to inputs.

Reply:
Oh then is it okay to leave the values like this? Or could you suggest a better way?

Reply:
I think it’s ok if your solution works. I just wanted to let you know potential reasons. Maybe you can find the time where you unintentionally assign inappropriate(undefined) values.